src/transformers/models/auto/tokenization_auto.py

```diff
@@ -43,153 +43,157 @@
 
 logger = logging.get_logger(__name__)
 
-
-TOKENIZER_MAPPING_NAMES = OrderedDict(
-    [
-        ("retribert", ("RetriBertTokenizer", "RetriBertTokenizerFast" if is_tokenizers_available() else None)),
-        ("roformer", ("RoFormerTokenizer", "RoFormerTokenizerFast" if is_tokenizers_available() else None)),
-        (
-            "t5",
+if TYPE_CHECKING:
+    # This significantly improves completion suggestion performance when
+    # the transformers package is used with Microsoft's Pylance language server.
+    TOKENIZER_MAPPING_NAMES: OrderedDict[str, Tuple[str, Optional[str]]] = OrderedDict()
```

Comment:
```suggestion
    TOKENIZER_MAPPING_NAMES: OrderedDict[str, Tuple[Optional[str], Optional[str]]] = OrderedDict()
```
First key is also optional if the slow tokenzier uses sentencepiece